src/app/app.component.ts

```diff
@@ -1,10 +1,135 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+
+import { bookType } from './book-type.enum';
+import { Author } from './models/author-model';
+import { Book } from './models/book-model';
+import { Bookstore } from './models/bookstore-model';
+import { Transaction } from './models/transaction-model';
+import { User } from './models/user-model';
+import { Wallet } from './models/wallet-model';
+import { BookstoreService } from './services/bookstore.service';
+import { UserService } from './services/user.service';
+import { transactionType } from './transaction-type.enum';
+import { Price } from './models/price-model';
+import { CurrencyType } from './currency-type.enum';
+import { Guid } from 'guid-typescript';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   title = 'bookstore-app';
+  public bookstore = new Bookstore();
+  public book1 = new Book(
+    Guid.create(),
+    'Angular 8',
+    bookType.Comedy,
+    new Author(Guid.create(), 'Janusz', 'Malarz'),
+    50,
+    10
+  );
+  public book2 = new Book(
+    Guid.create(),
+    'Alicja w Krainie Czarów',
+    bookType.Criminal,
+    new Author(Guid.create(), 'Janusz', 'Niewiem'),
```

Comment:
The same as in Book. You don't need to pass id

Reply:
ok